### PR TITLE
docs: spelling fixes in doc blocks

### PR DESCRIPTION
**Pull request by nandi95**
Just some grammar fixes my IDE highlighted

### CODE REVIEW

File: packages/router/src/RouterLink.ts [hunk 1]
```diff
@@ -65,7 +65,9 @@ export interface RouterLinkProps extends RouterLinkOptions {
    */
   exactActiveClass?: string
   /**
-   * Value passed to the attribute `aria-current` when the link is exact active. Defaults to "page"
+   * Value passed to the attribute `aria-current` when the link is exact active.
+   *
+   * @default 'page'
```
Comment (nandi95) [inline packages/router/src/RouterLink.ts:70]: I suspect there's more JSDoc improvements that can be made
https://jsdoc.app/tags-param.html#optional-parameters-and-default-values
but that can be done later

Comment (posva) [reply packages/router/src/RouterLink.ts:70]: We actually use tsdoc which can infer many things from types, so don't worry!
Can you change this to `defaultValue` instead? (https://tsdoc.org/pages/tags/defaultvalue/)

File: packages/router/src/history/common.ts [hunk 2]
```diff
@@ -74,8 +74,8 @@ export type ValueContainer<T> = { value: T }
 export interface RouterHistory {
   /**
    * Base path that is prepended to every url. This allows hosting an SPA at a
-   * subfolder of a domain like `example.com/subfolder` by having a `base` of
-   * `/subfolder`
+   * sub folder of a domain like `example.com/sub folder` by having a `base` of
```
Comment (posva) [inline packages/router/src/history/common.ts:77]: let's keep the one word version, it makes more sense

File: packages/router/src/history/hash.ts [hunk 3]
```diff
@@ -6,7 +6,7 @@ import { warn } from '../warning'
  * Creates a hash history. Useful for web applications with no host (e.g. `file://`) or when configuring a server to
  * handle any URL is not possible.
  *
- * @param base - optional base to provide. Defaults to `location.pathname + location.search` If there is a `<base>` tag
+ * @param {string} [base] - optional base to provide. Defaults to `location.pathname + location.search` If there is a `<base>` tag
```
Comment (posva) [inline packages/router/src/history/hash.ts:9]: no need for these, let's keep the old version

File: packages/router/src/history/memory.ts [hunk 4]
```diff
@@ -12,10 +12,10 @@ import {
 } from './common'
 
 /**
- * Creates a in-memory based history. The main purpose of this history is to handle SSR. It starts in a special location that is nowhere.
+ * Creates an in-memory based history. The main purpose of this history is to handle SSR. It starts in a special location that is nowhere.
  * It's up to the user to replace that location with the starter location by either calling `router.push` or `router.replace`.
  *
- * @param base - Base applied to all urls, defaults to '/'
+ * @param {string} [base=/] - Base applied to all urls
```
Comment (posva) [inline packages/router/src/history/memory.ts:18]: same

File: packages/router/src/matcher/pathParserRanker.ts [hunk 5]
```diff
@@ -52,23 +52,31 @@ export interface PathParser {
  */
 export interface _PathParserOptions {
   /**
-   * Makes the RegExp case sensitive. Defaults to false
+   * Makes the RegExp case-sensitive.
+   *
+   * @default false
```
Comment (posva) [inline packages/router/src/matcher/pathParserRanker.ts:57]: Same in this interface, let's use `defaultValue`

**Review by posva — CHANGES_REQUESTED** [review]
Thanks a lot!

**Review by posva — APPROVED** [review]

**Pull request merged**